src/transformState.js

```diff
@@ -61,8 +61,33 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  const linkToStateObj = state;
+
+  transforms.forEach((operationObj) => {
+    if (operationObj.operation === 'addProperties') {
+      for (const key in operationObj.properties) {
+        linkToStateObj[key] = operationObj.properties[key];
+      }
```

Comment:
```suggestion
      stateCopObj = {
        ...stateCopObj, ...properties,
      };
```
Можно использовать спред оператор для того что б добавить свойства.